modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -0,0 +1,246 @@
+:description: This page provides the syntax for the Cypher administration commands.
+[[administration-syntax]]
+= Cypher syntax for administration commands
+
+Almost all administration commands have variations.
+The most common are parts of the command that are optional or that can have multiple values.
+
+See below a summary of the syntax used to describe all versions of a command.
```

Comment:
would it make sense to have the table in its own subsection? looking at the preview and it feels odd to only see `database management command syntax` and `alias management command syntax` in the menu to the right and nothing for the `how to read these` parts